tests/Unit/AmpMath/amp_math_acos.cpp

```diff
@@ -33,9 +39,18 @@ int main(void) {
     gb[i] = fast_math::acos(ga[i]);
   }
 
-  float sum = 0;
+  _Tp sum = 0.0;
   for(unsigned i = 0; i < vecSize; i++) {
     sum += fast_math::fabs(fast_math::fabs(gc[i]) - fast_math::fabs(gb[i]));
```

Comment:
You're right, but we're doing this in every AmpMath test. I'll go through them and fix this.